docs/src/user-guide/getting-started.md

```diff
@@ -93,6 +92,57 @@ Your `.eslintrc.{js,yml,json}` configuration file will also include the line:
 
 Because of this line, all of the rules marked "(recommended)" on the [rules page](/docs/rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.
 
+## Global install
```

Comment:
We are using caps-case for subheadings throughout, so I think we should stick with that.
```suggestion
## Global Install
```